src/routes/Sponsors.svelte

```diff
@@ -139,22 +140,16 @@
 
 <div class="w-full px-8 py-28">
 	<div id="sponsors" class="container mx-auto flex flex-col items-center text-center max-w-5xl">
-		<h2 class="mb-8">Sponsor*innen</h2>
+		<h2 class="mb-8">{$_('sponsors.title')}</h2>
 
 		<div class="prose text-2xl text-left">
 			<p>
-				Der Non-Profit Verein «bernerit.rocks» organisiert den Anlass für Cloud Native
-				Praktiker*innen und setzt dabei auf Sponsor*innen. Durch Ihre Unterstützung präsentieren Sie
-				sich an vorderster Front der extrem stark wachsenden Cloud Native Community der Schweiz.
-				Werden Sie Sponsor*in und knüpfen Sie Kontakte mit den anwesenden Spezialist*innen am Cloud
-				Native Day! Bitte nehmen Sie via
-				<a href="mailto:sponsoring@cloudnativeday.ch">sponsoring@cloudnativeday.ch</a>
-				Kontakt mit uns auf.
+				{@html $_('sponsors.body')}
 			</p>
 		</div>
 
 		{#if goldSponsors.length > 0}
-			<h3 class="mt-24 w-full text-left">Gold</h3>
```

Comment:
Should we leave items we know are the same in every language directly in the HTML to keep duplication to a minimum?

Reply:
I would not consider it duplication, if a word happens to be the same in two languages. Instead, I would argue that we want all human-readable text centralized in the locale files and leave some sprinkled in the rest of the code.